Settings Page Design

mutahharaj opened the pull request:
- worked on layout to match website design
- centered and aligned the features
- rearranged features
- resized and reshaped buttons, titles, forms

mutahharaj committed:
Merge branch 'develop' into mutahhara_jannat

mutahharaj committed:
Merge branch 'develop' into mutahhara_jannat

mutahharaj committed:
Settings Page Design

- worked on layout to match website design
- centered and aligned the features
- rearranged features
- resized and reshaped buttons, titles, forms,

rishithab48 commented:
I think you can still enhance the design of the page? Some lines are longer than other and some buttons are not super consistent in size.

mutahharaj committed:
Update Settings.js

mutahharaj committed:
Additional Changes to Settings

mutahharaj committed:
Settings Updated

- aligned role selection and account visibility buttons
- shortened desire username input/text box
-

rishithab48 reviewed (approved):
Looks good to me. Make it more responsive.
